rotkehlchen/exchanges/kucoin.py

```diff
@@ -423,7 +430,10 @@ def _api_query_paginated(
                     )
                     continue
 
-                results.append(result)  # type: ignore
+                if isinstance(result, list):
+                    results.extend(result)
+                else:
+                    results.append(result)
```

Comment:
why is this needed? i'm guessing because `deserialize_asset_movement()` returns a list now?

Reply:
Yes, its due to asset movements now being in a list here. I expect eventually when we convert trades to being based on history events as well they will also need to be in a list due to having fees as separate events and then this will be changed to just `results.extend(result)`

Reply:
can you then add a comment with a TODO here for this explaining it?